### PR TITLE
Fixed #32355 -- Dropped support for Python 3.6 and 3.7

### PR DESCRIPTION
**Pull request by felixxm**
ticket-32355

### CODE REVIEW

File: django/db/migrations/questioner.py [hunk 1]
```diff
@@ -44,7 +44,6 @@ def ask_initial(self, app_label):
         except ImportError:
             return self.defaults.get("ask_initial", False)
         else:
-            # getattr() needed on PY36 and older (replace with attribute access).
```
Comment (wkschwartz) [inline django/db/migrations/questioner.py:47]: Thanks for not heeding the comment! I came here just to check on it... (Cf. #13841, specifically [this](https://github.com/django/django/pull/13841/commits/34424973507422b05cd6f1bf0cd51d874a46d182#diff-17c5ace9fb3dcbc1f10671d7fd7d248bc394020ce095b0c6a94c8a163a8d4a54L47))

**Review by carltongibson — APPROVED** [review]
Hey @felixxm, thank you for this. Super! 

Let's progress 😉

**Comment by felixxm** [comment]
@carltongibson Thanks :+1: Rebased from master.

**Comment by felixxm** [comment]
buildbot, test on selenium.

**Comment by felixxm** [comment]
buildbot, test on oracle.

File: django/utils/version.py [hunk 2]
```diff
@@ -9,8 +9,6 @@
 # or later". So that third-party apps can use these values, each constant
 # should remain as long as the oldest supported Django version supports that
 # Python version.
-PY36 = sys.version_info >= (3, 6)
```
Comment (timgraham) [inline django/utils/version.py:12]: Following the comment's instructions, PY36 shouldn't be removed until Django 2.2 (which supports Python 3.5) is no longer supported. PY37 can be removed when Django 3.2 (which supports Python 3.6) is no longer supported.

Comment (carltongibson) [reply django/utils/version.py:12]: Thanks @timgraham! #13996.

**Pull request merged**